jwst/datamodels/container.py

```diff
@@ -381,6 +381,50 @@ def close(self):
             for model in self._models:
                 model.close()
 
+    @property
+    def crds_observatory(self):
+        """
+        Get the CRDS observatory for this container.  Used when selecting
+        step/pipeline parameter files when the container is a pipeline input.
+
+        Returns
+        -------
+        str
+        """
+        # Eventually ModelContainer will also be used for Roman, but this
+        # will work for now:
+        return "jwst"
```

Comment:
Since `ModelContainer` subclasses `JwstDataModel`, is this needed?

Reply:
This was in anticipation of your project to make `ModelContainer` independent of `DataModel`.  We're going to need a separate method here anyway since eventually we should pull this from one of the member models, but I didn't want to do that right away since at the moment that requires opening the model a second time.

Reply:
This is a more general comment but I didn't see how to put it at the top. This is all being done with data models in the JWST package. How do you see this merging with stdatamodels? (Sorry if you answered this elsewhere.)

Reply:
@jdavies-st is working on moving `ModelContainer` to stdatamodels.  Once that's done we may or may not need a JWST-specific ModelContainer subclass.  If we do we can put this hard-coded "jwst" in there, if we don't then we'll have to retrieve the observatory code from one of the container's member models.